cmd/internal/switcher/sonic/redis/applier.go

```diff
@@ -129,7 +130,9 @@ func (a *Applier) configureUnprovisionedPort(interfaceName string, isUp bool) er
 		return fmt.Errorf("failed to update Port info for interface %s: %w", interfaceName, err)
 	}
 
-	return a.ensureInterfaceIsVlanMember(ctx, interfaceName, "Vlan4000")
+	vlan := "Vlan" + strconv.FormatUint(uint64(pxeVlanID), 10)
```

Comment:
Maybe move it between line 58 and 59 and provide the whole vlan name to configureUnprovisionedPort.